src/components/Facility/CentralNursingStation.tsx

```diff
@@ -43,9 +43,20 @@ interface Props {
 export default function CentralNursingStation({ facilityId }: Props) {
   const { t } = useTranslation();
   const [isFullscreen, setFullscreen] = useFullscreen();
+  const [PER_PAGE_LIMIT, setPerPageLimit] = useState(6);
```

Comment:
@shivankacker ,is this okay now?

Reply:
like already said in previous PR's review, use `useBreakpoints` instead. please do not skip reviews

Reply:
cc: @bodhish